### DISCUSSION TITLE
Diagram becomes dirty even though we have not done changes

### DISCUSSION THREAD

**Discussion by sathya-1994** [eclipse-glsp/glsp#765] [Q&A] [answered]
I am using GLSP's vscode extension.

Steps to reproduce:
1. Open diagram form .wf file
2.  Try to edit node label using double
3.  Don't update label click on other node

Expected: Diagram editor should not be come dirty
Actual: Diagram editor will be dirty even though we have not changed content

**Comment by planger** [comment, marked as answer]
I believe this is a bug. I'll convert this into a Github issue.

**Reply by planger** [reply]
https://github.com/eclipse-glsp/glsp/issues/766
